plugins/hydra_nevergrad_sweeper/tests/test_nevergrad_sweeper_plugin.py

```diff
@@ -23,25 +25,22 @@ def test_discovery() -> None:
 
 
 @pytest.mark.parametrize(  # type: ignore
-    "description,param_cls,value_cls",
+    "override,param_cls,value_cls",
```

Comment:
this function tested the conversion from a string to a resulting parameter for Nevergrad.
Don't change the meaning of the test. input should still be a string.